papermill/execute.py

```diff
@@ -229,17 +229,52 @@ def _find_parameters_index(nb):
         raise PapermillException("Multiple parameters tags found")
     return parameters_indices[0]
 
-def _form_escaped_value(str_val):
+def _translate_escaped_str(str_val):
+    '''Reusable by most interpreters'''
```

Comment:
Style tip: use triple double quotes for docstrings `"""`

Reply:
Opps, yep, fixing.